assets/wire/components/WireListItem.jsx

```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames';
 import {get} from 'lodash';
 
 import {gettext, fullDate, wordCount, LIST_ANIMATIONS, getSlugline} from 'utils';
-import {getPicture, getThumbnailRendition, showItemVersions, shortText, isKilled, getVideos} from 'wire/utils';
+import {getPicture, getPreviewRendition, showItemVersions, shortText, isKilled, getVideos} from 'wire/utils';
```

Comment:
why do you need this change? I don't think we should do this, thumbnail rendition is configured not to include watermark, while preview could have it.

Reply:
Because thumbnail rendition is cutting images to 640x480 and we need original aspect ratio. In that case please make a logic for generating this kind of rendition, which maintains original image aspect ratio.

Reply:
that was just temporary, new images should have original aspect ration for thumbnails

Reply:
Ok thanks for clarifying, I left only css part.